CMake/BuildFlatBuffers.cmake

```diff
@@ -264,14 +282,14 @@ function(flatbuffers_generate_headers)
       ${include_params}
       -c ${schema}
       ${FLATBUFFERS_GENERATE_HEADERS_FLAGS}
-      DEPENDS ${FLATC_TARGET} ${schema}
+      DEPENDS ${FLATC} ${schema}
       WORKING_DIRECTORY "${working_dir}"
       COMMENT "Building ${schema} flatbuffers...")
     list(APPEND all_generated_header_files ${generated_include})
     list(APPEND all_generated_source_files ${generated_source_file})
-    list(APPEND generated_custom_commands "${generated_include}" "${generated_source_file}")
+    list(APPEND all_generated_files "${generated_include}" "${generated_source_file}")
```

Comment:
Renamed generated_custom_commands -> all_generated_files because the old name was very misleading